<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>398</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔스퀘어OTF_ac Light</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>모두의 이유식</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>food.png</normalon>
   </iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어OTF_ac Light</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>이유식 찾기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Searchline">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radio_first">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>초기 이유식</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_second">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>중기 이유식</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_third">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>후기 이유식</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
